app/src/main/java/com/dragic/gamehunter/model/DealResponse.kt

```diff
@@ -0,0 +1,12 @@
+package com.dragic.gamehunter.model
```

Comment:
Will this be the class that holds the JSON API response? If yes, it should be moved into the `networking` package. API response data classes are transient and unstable, which is why they shouldn't be used in the rest of the codebase. It's just a mechanism to get data from the API and transform it into meaningful business classes (often called entities) that the rest of the app can use safely and conveniently :)